packages/nextjs/src/utils/config.ts

```diff
@@ -1,10 +1,37 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-var-requires */
 /* eslint-disable @typescript-eslint/no-explicit-any */
-// import { version as nextVersion } from './node_modules/next/package.json';
 import { getSentryRelease } from '@sentry/node';
 import { logger } from '@sentry/utils';
 import defaultWebpackPlugin, { SentryCliPluginOptions } from '@sentry/webpack-plugin';
 import * as SentryWebpackPlugin from '@sentry/webpack-plugin';
 import * as fs from 'fs';
+import * as path from 'path';
+
+/**
+ * Starting at `startPath`, move up one directory at a time, searching for `searchFile`.
+ *
+ * @param startPath The location from which to start the search.
+ * @param searchFile The file to search for
+ * @returns The absolute path of the file, if it's found, or undefined if it's not.
+ */
+function findUp(startPath: string, searchFile: string): string | undefined {
+  if (!fs.existsSync(startPath)) {
+    throw new Error(`The given \`startPath\` value (${startPath}) does not exist.`);
+  }
+
+  let currentDir = fs.statSync(startPath).isFile() ? path.dirname(startPath) : startPath;
+  while (currentDir !== '/') {
+    const possiblePath = path.join(currentDir, searchFile);
+    if (fs.existsSync(possiblePath)) {
+      return possiblePath;
+    }
+
+    currentDir = path.join(currentDir, '..');
```

Comment:
```suggestion
    currentDir = path.resolve(currentDir, '..');
```

This will transform `/foo/bar/baz` into `/foo/bar`, `/foo`, `/` instead of `/foo/bar/baz/..`, `/foo/bar/baz/../..`, `/foo/bar/baz/../../..`